app/src/usb.c

```diff
@@ -38,6 +38,22 @@ static const struct hid_ops ops = {
     .int_in_ready = in_ready_cb,
 };
 
+#ifdef CONFIG_ZMK_USB_REPORT_LEDS
+
+int zmk_usb_hid_receive_report(uint8_t *report, size_t len) {
+    k_sem_take(&hid_sem, K_MSEC(30));
+    int err = hid_int_ep_read(hid_dev, report, len, NULL);
```

Comment:
Is the `ret_bytes` parameter to `hid_int_ep_read()` not necessary? I would expect calling code to need to be able to check how many bytes were actually read, since it might be smaller than `len`.